gstd/gstd_parser.c

```diff
@@ -161,7 +161,7 @@ gstd_parser_parse_raw_cmd (GstdSession * session, gchar * action, gchar * args,
 
   // Alias the empty string to the base
   if (!uri)
-    uri = "/";
+    uri = (gchar*)"/";
```

Comment:
Let's make `uri` a `const` instead.

Reply:
`uri` can be declarated as `const gchar` pointer, but in the context of the function, the variable can be initialized with a pointer to a constant string or with a pointer to a normal string, based on this I think it should be kept as a normal pointer.